src/cli/output/to-mbit.js

```diff
@@ -0,0 +1,7 @@
+export const toMbit = (kbit) => {
+  if (typeof kbit === 'string') {
+    kbit = parseFloat(kbit)
+  }
+  return Math.round(kbit/1024*100)/100
+}
+export default toMbit
```

Comment:
In NDT code I'm using kbit/s because that is what the NDT spec says. Here the ideal would in theory be to scale the result such as is in the range 0-1,000 and then apply the proper scaling factor.

Reply:
Probably. We can always tune these UI things later once we have a solid design for it. Should not be hard since I am following a pattern of routing all rendering stuff through `src/cli/output` functions.

Reply:
Yap, that was not meant as a blocker comment but rather as an observation for the future!